Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.m

```diff
@@ -144,8 +146,9 @@ - (void) adjustFrame
         
         CGRect buttonsFrame = _buttonsView.frame;
         buttonsFrame.origin.y = f.size.height - self.buttonsViewHeight - bottomMargin;
-        buttonsFrame.size.height = self.buttonsViewHeight + bottomMargin;
+        buttonsFrame.size.height = self.buttonsViewHeight;
         _buttonsView.frame = buttonsFrame;
+        _buttonsViewHeightConstraint.constant = self.buttonsViewHeight;
```

Comment:
Not sure you need to introduce a constraint here. Manual frame height calculation works ok here